src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -349,7 +349,7 @@ public static Module parseModule(String code, Optional<String> name) throws Pars
      * @throws ParseException if the args string is invalid.
      */
     public static Type parseFirstArg(String args, String errorMessage) throws ParseException {
-        String firstArg = args.split(" ", 2)[0];
+        String firstArg = args.trim().split(" ", 2)[0];
```

Comment:
Yeah this is a good change. Was thinking if I should add it but you can merge it in when yours done